src/luarocks/test/command.lua

```diff
@@ -32,14 +32,22 @@ function command.run_tests(test, args)
       end
    end
    
+   local ok, err
+
    if test.script then
       if not fs.exists(test.script) then
          return nil, "Test script " .. test.script .. " does not exist"
       end
       local lua = fs.Q(dir.path(cfg.variables["LUA_BINDIR"], cfg.lua_interpreter))  -- get lua interpreter configured
-      return fs.execute(lua, test.script, unpack(args))
+      ok, err = fs.execute(lua, test.script, unpack(args))
```

Comment:
As `fs.execute` never returns an error message, you don't need to try and get `err` from it.